### ISSUE TITLE
How to use NchanSubscriber?

### ISSUE THREAD

**Issue by ntv-wangjian** [slact/nchan#179]
Hello,
How to use NchanSubscriber.js in a html page?

**Comment by slact** [comment]
Take a look at the comments in the [file](https://github.com/slact/nchan/blob/master/NchanSubscriber.js). I will also be adding a demo in a few days using NchanSubscriber.js.

**Comment by ntv-wangjian** [comment]
Thanks  a lot!  I include this js file into a web page, and use it as the direction of the comments , but a lot of errors accur.

**Comment by slact** [comment]
You're going to need to be more specific than that if you want me to be able to help you out...

**Comment by seblau** [comment]
I get the following error. The name variable does contain websocket in this case.

My NchanSubscriber usage is like the following:

    var sub = NchanSubscriber('http://10.0.0.12/pub', 'websocket');
    sub.on("message", function(message, message_metadata) {alert(message);});

![screen shot 2016-05-21 at 19 13 07](https://cloud.githubusercontent.com/assets/3884582/15449787/2fd94040-1f88-11e6-9edb-c485b867ffea.png)

**Comment by slact** [comment]
oh, my apologies, I had incorrectly documented the usage. You need to use the  `new` keyword:
```js
var sub = new NchanSubscriber('http://10.0.0.12/pub', 'websocket');
sub.on("message", function(message, message_metadata) {alert(message);});
```

**Comment by ntv-wangjian** [comment]
My usage:
var url = "http://192.168.1.200:8000/msg_reader/1";
var sub = new NchanSubscriber(url, 'longpoll');
	sub.on("message", function(message, message_metadata) {
		  alert(message); 
    });
sub.start();

Get an error as:
**NchanSubscriber.js:212 Uncaught ReferenceError: headers is not defined**

**Comment by slact** [comment]
@ntv-wangjian : Ah, my bad. fixed in 672b402a8f98cc8db90d7c4e86a9767e12525858 . Grab it from master and give it a try.

**Comment by ntv-wangjian** [comment]
@slact It‘s OK now, Thanks a lot!
